hibernate-core/src/main/java/org/hibernate/boot/model/process/spi/MetadataBuildingProcess.java

```diff
@@ -311,45 +314,34 @@ public void finishUp() {
 		return metadataCollector.buildMetadataInstance( rootMetadataBuildingContext );
 	}
 
-//	private static JandexInitManager buildJandexInitializer(
```

Comment:
Can we leave this in place?  Possibly with a `todo (7.0)` comment?

Reply:
Added back.